runtime-config-linux.md

```diff
@@ -45,6 +45,30 @@ container via system level IPC.
 * **user** the container will be able to remap user and group IDs from the host to local users and groups
 within the container.
 
+## User namespace mappings
+
+uid/gid mappings describe the user namespace mappings from the host to the container.
+The mappings represent how the bundle `rootfs` expects the user namespace to be setup and the runtime SHOULD NOT modify the permissions on the rootfs to realize the mapping.
+*hostID* is the starting uid/gid on the host to be mapped to *containerID* which is the starting uid/gid in the container and *size* refers to the number of ids to be mapped.
+There is a limit of 5 mappings which is the Linux kernel hard limit.
+
+```json
```

Comment:
On Sun, Sep 13, 2015 at 07:34:11PM -0700, Gao feng wrote:
> @wking You mean we should add `Example` before all of json examples
> in this file?

Yes, although we can leave examples you aren't touching for follow-up
PRs (although it looks like you got a lot of them in 107a454a0, and
I'm fine with that too).